index.html

```diff
@@ -9,7 +9,7 @@
 </head>
 
 <body>
-
+    <h1>Welcome to Potonify!! Enjoy you stay...</h1>
```

Comment:
Could you remove the 'Enjoy you stay' string and make the 'Welcome to Potonify' string red.